packages/valaxy-theme-yun/node/index.ts

```diff
@@ -1,5 +1,5 @@
+import type { ResolvedValaxyOptions, ValaxyThemeOptions } from 'valaxy/node'
```

Comment:
`valaxy/node` is a solution to avoid confusion.

However, node is exported by default, and for concision, I prefer to eat 'valaxy' directly.